src/game/level_update.c

```diff
@@ -1329,7 +1325,11 @@ s32 init_level(void) {
     }
 
 #ifdef PUPPYPRINT_DEBUG
-    gLoadLevelAreaTime = osGetCount() - first;
+    if (gInitLevelTime) {
+        u32 totalTime = osGetCount() - gInitLevelTime;
+        append_puppyprint_log("Level loaded in %2.3fs.", (f64) (f32)((totalTime) / 46875000.0f));
```

Comment:
Please use something like `OS_CYCLES_TO_USEC` rather than this magic constant.

Reply:
Constant was removed and replaced with `OS_CYCLES_TO_USEC`